pkg/kubelet/dockershim/docker_checkpoint.go

```diff
@@ -73,7 +75,8 @@ type PodSandboxCheckpoint struct {
 	Checksum checksum.Checksum `json:"checksum"`
 }
 
-func NewPodSandboxCheckpoint(namespace, name string, data *CheckpointData) DockershimCheckpoint {
+// NewPodSandboxCheckpoint int a PodSandboxCheckpoint with the given args
```

Comment:
Nit: does `int` mean `initializes` here? I think using the full word will help make this more readable.

Reply:
Sorry, it's a typo, I'll fix it